### ISSUE TITLE
[CST] [DDL]  Add 5103 letter(s) to DDL

### ISSUE THREAD

**Issue by jacobworrell** [department-of-veterans-affairs/va.gov-team#72464]
## Value Proposition

As a veteran with a pending disability claim, I want to know my 5103 status, so I can either waive my right to 30 day evidence gathering period or stay informed about actions I've taken to submit evidence. 

## Background

Here are the 5103 letters that are created in VBMS:

<img width="1395" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBM3ArQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--e186bc96557efa04769ed8924fbbd3627408e3fd/image.png" alt="image.png" />

Edit: We've also added "Custom 5103" doc id 858 to the requirements.

## Tasks

- Update code to allow for differentiating the language used for the filename downloaded. (this actually should become a new ticket. Leaving the note here as a reminder for me to write that ticket. cc: @jacobworrell)

## Open Questions

- [x] Which 5103 notices are important to send to veterans and why? 

- Standard 5103 notice - acknowledges what the veteran can return to prevent VBA from waiting 30 days for information.
- 5103/ DTA letter

- [x] Out of all the 5103 notices we could show in CST, what is the priority for each and why?

- We want to show both the above letters. Standard 5103 Notice (doc ID 704) and 5103/DTA Letter (doc ID 706) and Custom 5103 (858). These notices inform the veteran when they could take an action to expedite their claim and/or notifies them of their current 5103 status.

## Acceptance Criteria

- [x] Doc types with ids 704, 706 and 858 should be presented to the end user as "5103 Notice" as the title of the document. 
- [x] Letters added to DDL continue to sort using reverse chronological logic (most recent letters on top). 
- [x] 5103 docs added to the letters list should leverage the existing metadata such that we include the same details we're including for decision letters when possible.
- [x] Must be behind a new feature flag.

**Comment by SilkeDannVA** [comment]
PRs being held up by platform issue.
